traits/tests/test_trait_list_dict.py

```diff
@@ -20,7 +20,7 @@
 class A(HasTraits):
     alist = List(Int, list(sm.range(5)))
     adict = Dict(Str, Int, dict(a=1, b=2))
-    aset = Set(Int, list(sm.range(5)))
+    aset = Set(Int, set(sm.range(5)))
```

Comment:
This was a bug in the test, I assume (unless we're being particularly perverse).